samples/features/sql-big-data-cluster/connectivity/webhdfs-java-client/src/main/java/com/microsoft/mssql/App.java

```diff
@@ -0,0 +1,118 @@
+                                System.out.println(response.getStatusLine());
+                                System.out.println("----------------------------------------");
+                                if (entity != null) {
+                                        System.out.println(EntityUtils.toString(entity));
+                                }
+                                System.out.println("----------------------------------------");
+                                EntityUtils.consume(entity);
+                                // Second request to put the content to that location
+                                File testUploadFile = new File(inputFilePath);
+                                HttpEntity putData = MultipartEntityBuilder.create().addBinaryBody("upfile", testUploadFile).build();
+                                HttpUriRequest putRequest = RequestBuilder.put(newlocation).setEntity(putData).build();//setEntity(postData).build();
+                                System.out.println("Executing request " + putRequest.getRequestLine());
+                                CloseableHttpResponse response2 = client2.execute(putRequest);
+                                HttpEntity entity2 = response2.getEntity();
+                                System.out.println("----------------------------------------");
+                                System.out.println(response2.getStatusLine());
+                                System.out.println("----------------------------------------");
+                                if (entity2 != null) {
+                                        System.out.println(EntityUtils.toString(entity2));
+                                }
+                                System.out.println("----------------------------------------");
+                                
+                        }
+                } catch (Exception e) {
+                	System.out.println(e);
+                }
+	
+        }
+ 
+	public static void main(String[] args) throws Exception {
+		// These properties can be specified using command line as well by using 
+		// -Djava.security.auth.login.conf=login.conf -Djava.security.krb5.realm=MYDOMAIN.LOCAL
+		// In case command line is used to pass these arguments, then code below setting system properties should not be used..
+		System.setProperty("java.security.auth.login.config", LOGIN_CONF_PATH); // The login conf for Java GSS to use. See readme for a sample.
+		System.setProperty("java.security.krb5.realm", DOMAIN_REALM); 	// The domain name.
+		System.setProperty("java.security.krb5.kdc", KDC); // Provide the KDC name here
+		System.setProperty("sun.security.krb5.debug", "true"); // This can be set to true to print debug information about Kerberos login.
+		System.setProperty("javax.security.auth.useSubjectCredsOnly", "false");
+	        
+                for(int i = 0; i < args.length; i++) { 
```

Comment:
Indentation of this for loop is incorrect. Please use a formatter in Eclipse or your fav IDE.

Reply:
There should be some validation done on the number of input args. The code should provide guidance about the "required" arguments and fail when the number of input arguments don't match the number of required arguments.